moco/src/main/java/com/cognifide/aem/stubs/moco/MocoStubs.java

```diff
@@ -83,13 +101,49 @@ protected void deactivate() {
 
   private void start() {
     LOG.info("Starting AEM Stubs Moco Server");
+
+    server = createServer();
+    runner = runner(server);
+    runner.start();
+  }
+
+  private ActualHttpServer createServer() {
+    AtomicReference<ActualHttpServer> result = new AtomicReference<>();
```

Comment:
intellij suggested that ... to mitigate java limitation - assigning vars outside lambda